core/atomic/string.go

```diff
@@ -0,0 +1,23 @@
+package atomic
```

Comment:
same, why not using directly atomic Value?

Reply:
Same as above. It's important to encapsulate and hide the `casting to string` logic. Improved readability, better semantics, less bugs.